guides/v2.1/coding-standards/docblock-standard-javascript.md

```diff
@@ -94,7 +96,7 @@ function&nbsp;someMethod(obj)&nbsp;{
 }
 </pre>
 
-<h3 id="frontend-dev-guide-javascript-js-coding-docblock-propertycomments">Property comments</h3>
+### Property comments
```

Comment:
Please append `{#frontend-dev-guide-javascript-js-coding-docblock-propertycomments}` to preserve the custom id.